<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <arg name="bag_file_path" />
    <arg name="topics" />
    <arg name="output" default="screen"/>

	<node name="$(anon recorder)" pkg="rosbag" type="record" args="--lz4 -O $(arg bag_file_path) $(arg topics)" output="$(arg output)" />

</launch>
